core/test/object-manipulator.ts

```diff
@@ -121,4 +125,31 @@ definitions:
       assert.strictEqual(resultObject.paths['/api/circular'].get.description, 'FUN TIME');
     }
   }
+
+  @test async 'skip-transform-failure'() {
```

Comment:
Ahhh, somehow I blanked out on the fact that this is the test name, not an indicator that the test is skipped :)  This name is better, thanks!